proptest-derive/src/interp.rs

```diff
@@ -69,62 +69,53 @@ fn parse_lit_int(mut s: &str) -> Option<u128> {
 }
 
 /// Parse a suffix of an integer literal.
-fn parse_suffix(lit: &str) -> IS {
+fn parse_suffix(lit: &str) -> Option<&'static str> {
     [
-        ("i8", IS::I8),
-        ("i16", IS::I16),
-        ("i32", IS::I32),
-        ("i64", IS::I64),
-        ("i128", IS::I128),
-        ("isize", IS::Isize),
-        ("u8", IS::U8),
-        ("u16", IS::U16),
-        ("u32", IS::U32),
-        ("u64", IS::U64),
-        ("u128", IS::U128),
-        ("usize", IS::Usize),
+        "i8", "i16", "i32", "i64", "i128", "isize", "u8", "u16", "u32", "u64",
+        "u128", "usize",
     ]
     .iter()
-    .find(|&(s, _)| lit.ends_with(s))
-    .map(|(_, suffix)| suffix.clone())
-    .unwrap_or(IS::None)
+    .find(|s| lit.ends_with(*s))
+    .copied()
 }
 
 /// Interprets an integer literal in a string.
 fn eval_str_int(lit: &str) -> Option<u128> {
-    use std::{i128, i16, i32, i64, i8, u16, u32, u64, u8};
-    use syn::IntSuffix::*;
-
     let val = parse_lit_int(lit)?;
-    Some(match parse_suffix(lit) {
-        None => val,
-        I8 if val <= i8::MAX as u128 => val,
-        I16 if val <= i16::MAX as u128 => val,
-        I32 if val <= i32::MAX as u128 => val,
-        I64 if val <= i64::MAX as u128 => val,
-        U8 if val <= u128::from(u8::MAX) => val,
-        U16 if val <= u128::from(u16::MAX) => val,
-        U32 if val <= u128::from(u32::MAX) => val,
-        U64 if val <= u128::from(u64::MAX) => val,
-        Usize if val <= u128::max_value() => val,
-        Isize if val <= i128::max_value() as u128 => val,
-        U128 => val,
-        I128 if val <= i128::MAX as u128 => val,
+    let checked_val = if let Some(suffix) = parse_suffix(lit) {
+        match suffix {
+            "i8" if val <= i8::MAX as u128 => val,
+            "i16" if val <= i16::MAX as u128 => val,
+            "i32" if val <= i32::MAX as u128 => val,
+            "i64" if val <= i64::MAX as u128 => val,
+            "u8" if val <= u128::from(u8::MAX) => val,
+            "u16" if val <= u128::from(u16::MAX) => val,
+            "u32" if val <= u128::from(u32::MAX) => val,
+            "u64" if val <= u128::from(u64::MAX) => val,
+            "usize" if val <= usize::MAX as u128 => val,
+            "isize" if val <= isize::MAX as u128 => val,
```

Comment:
Wasn't sure if these were intentionally being treated as `(i|u)128` for a reason, or if it was a copy-paste style error